stat.go

```diff
@@ -1,6 +1,7 @@
 package fsutil
 
 import (
+	gofs "io/fs"
```

Comment:
Curious; was there a conflict somewhere that made an alias needed? (Otherwise, I think it'd be good to avoid aliases)

Reply:
Actually wondering if the change is needed, as `os.FileInfo` is an alias for `fs.FileInfo` (and `os.FileInfo` has not been deprecated 🤔  https://github.com/golang/go/blob/1a7709d6af76c06d465c5e969b502fc206f8e687/src/os/types.go#L21

```go
// A FileInfo describes a file and is returned by Stat and Lstat.
type FileInfo = fs.FileInfo
```

Reply:
ya, there's a conflict with the `fs` type in this package:
https://github.com/tonistiigi/fsutil/blob/9e7a6df4857652bc85225f55e2edb8aece3ecd43/fs.go#L29
https://github.com/tonistiigi/fsutil/blob/9e7a6df4857652bc85225f55e2edb8aece3ecd43/diskwriter.go#L7

but you're right, they're the same, i'll change it back.